index.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="./main.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NYTarticle</title>
+</head>
+
+<body>
+    <header class="flex">
+        <div id="left-header" class="inline-flex">
+            <button><img src="./hamburger-menu-lines-icon.png" alt="burger-menu"></button>
+            <button id="search-button"><img src="./index.png" alt="search-button"></button>
+            <p id="hidden-p">SPACE & COSMOS</p>
+        </div>
+
+        <div id="mid-header">
+            <img src="./ny-times-logo.png" alt="main-header-logo">
```

Comment:
### Navigation bar
- [ ] Kindly ensure The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)). Currently, we have an image which is a good thing. Do you mind turning it into a link?